### PR TITLE
Fix checkpatch errors and warnings

### PR DESCRIPTION
**Pull request by stsoe**

### CODE REVIEW

File: src/runtime_src/driver/xclng/drm/xocl/subdev/mb_scheduler.c [hunk 1]
```diff
@@ -37,7 +37,7 @@
  *  - loops repeatedly when there is work to do
  *  - moves pending commands into a scheduler command queue
  *
- * [new -> pending]. The xocl API adds exec BOs to KDS.  The exec BOs are
+ * [new -> pending]. The xocl API adds exec BOs to KDS.	 The exec BOs are
```
Comment (maxzhen) [inline src/runtime_src/driver/xclng/drm/xocl/subdev/mb_scheduler.c]: really? what is the rule we're following here?

File: src/runtime_src/driver/xclng/drm/xocl/subdev/mb_scheduler.c [hunk 2]
```diff
@@ -2402,56 +2428,59 @@ static void destroy_client(struct platform_device *pdev, void **priv)
...
 	unsigned int	loops = 0;
 	int pid = pid_nr(task_tgid(current));
-	unsigned bit;
+	unsigned int bit;
 	struct ip_layout *layout = XOCL_IP_LAYOUT(xdev);
 
 	bit = layout
-		? find_first_bit(client->cu_bitmap, layout->m_count)
-		: MAX_CUS;
+	  ? find_first_bit(client->cu_bitmap, layout->m_count)
```
Comment (maxzhen) [inline src/runtime_src/driver/xclng/drm/xocl/subdev/mb_scheduler.c]: not required to use tab for indent?

File: src/runtime_src/driver/xclng/drm/xocl/subdev/mb_scheduler.c (hunk 1 above)
Comment (stsoe) [reply src/runtime_src/driver/xclng/drm/xocl/subdev/mb_scheduler.c]: Maybe an editor setting?  I think this came from the merge from Sonal's repo.

File: src/runtime_src/driver/xclng/drm/xocl/subdev/mb_scheduler.c (hunk 2 above)
Comment (stsoe) [reply src/runtime_src/driver/xclng/drm/xocl/subdev/mb_scheduler.c]: Guess not.  It's part of the expression, not a new statement; maybe that's why.

**Pull request merged**